.github/workflows/e2e.yml

```diff
@@ -17,5 +17,6 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
     - name: E2E Test
+      shell: bash
       run: |
-        make e2e
+        sudo env "PATH=$PATH" bash ./test/e2e/run_test.sh
```

Comment:
why set `env "PATH=$PATH"`

Reply:
and why not make e2e

Reply:
If run `make e2e` directly, there are permission problems.
![image](https://github.com/kmesh-net/kmesh/assets/140870529/d2f5d049-d2b1-4b7b-a785-362a59303090)

Set `env "PATH=$PATH"` is refer to https://github.com/kmesh-net/kmesh/blob/da515ea41ea5464992e4cd22140bf28b624f0536/.github/workflows/main.yml#L30

Since the e2e tests also require `Build Kmesh`, it is also straightforward to set up env to run the e2e scripts.

Reply:
I guess sudo make e2e works.

Reply:
done